buildozer/__init__.py

```diff
@@ -309,15 +309,21 @@ def cmd(self, command, **kwargs):
                 if get_stdout:
                     ret_stdout.append(chunk)
                 if show_output:
-                    stdout.write(chunk.decode('utf-8', 'replace'))
+                    try:
+                        stdout.write(chunk.decode('utf-8', 'replace'))
+                    except UnicodeEncodeError:
+                        stdout.write('?' * len(chunk))
             if fd_stderr in readx:
                 chunk = process.stderr.read()
                 if not chunk:
                     break
                 if get_stderr:
                     ret_stderr.append(chunk)
                 if show_output:
-                    stderr.write(chunk.decode('utf-8', 'replace'))
+                    try:
+                        stderr.write(chunk.decode('utf-8', 'replace'))
+                    except UnicodeEncodeError:
+                        stderr.write('?' * len(chunk))
```

Comment:
I have no idea why we're doing this, could you elaborate? Or maybe write a test case for it?

Reply:
I got that error on my Ubuntu ...    It seems to me that in some cases sub-process that is executed by `buildozer` to run compile/build steps is returning non-ascii characters to `stdout` (and could be potentially to `stderr` as well) .   I do not remember which of the packages it was failing but that fix made it working.  probably not nicest solution, but at least it is a safe operation.